src/wasm-lib/kcl/src/std/sweep.rs

```diff
@@ -104,15 +92,25 @@ pub async fn sweep(exec_state: &mut ExecState, args: Args) -> Result<KclValue, K
 /// // Create a spring by sweeping around the helix path.
 /// springSketch = startSketchOn('YZ')
 ///     |> circle({ center = [0, 0], radius = 1 }, %)
-///     |> sweep({ path = helixPath }, %)
+///     |> sweep(path = helixPath)
 /// ```
 #[stdlib {
     name = "sweep",
     feature_tree_operation = true,
+    keywords = true,
+    unlabeled_first = true,
+    args = {
+        sketch = { docs = "The sketch that should be swept in space" },
+        path = { docs = "The path to sweep the sketch along" },
+        sectional = { docs = "If true, the sweep will be broken up into sub-sweeps (extrusions, revolves, sweeps) based on the trajectory path components." },
```

Comment:
Just a note here: sectional is missing from point-and-click impl today, create an issue to track it https://github.com/KittyCAD/modeling-app/issues/5301 (v1 scope creep candidate labeled ofc!)